course/src/main/java/org/openedx/course/presentation/outline/CourseOutlineUIState.kt

```diff
@@ -8,7 +8,10 @@ sealed class CourseOutlineUIState {
     data class CourseData(
         val courseStructure: CourseStructure,
         val downloadedState: Map<String, DownloadedState>,
-        val resumeComponent: Block?
+        val resumeComponent: Block?,
+        val courseSections: Map<String, List<Block>>,
+        val courseSectionsState: Map<String, Boolean>,
+        val downloadsCount: Map<String, Int>
```

Comment:
We can take the same decision we will take on CourseVideosUIState. Plus the naming convention needs to be improved. The `courseSections` are storing the subSections and `downloadsCount` should mention subSection in its name.